src/NumericalAlgorithms/DiscontinuousGalerkin/FluxCommunicationTypes.hpp

```diff
@@ -95,11 +95,11 @@ struct FluxCommunicationTypes {
     using temporal_id = db::item_type<typename Metavariables::temporal_id>;
     using type = std::map<
         temporal_id,
-        std::unordered_map<
-            std::pair<Direction<volume_dim>, ElementId<volume_dim>>,
-            std::pair<temporal_id, PackagedData>,
-            boost::hash<
-                std::pair<Direction<volume_dim>, ElementId<volume_dim>>>>>;
+        FixedHashMap<2 * volume_dim,
```

Comment:
Couldn't there be more fluxes received than `2 * volume_dim`? Should this be `maximum_number_of_neighbors`? If this is correct, a comment would help.

Reply:
I think you're right. With AMR you'd possibly receive data from 24 neighbors.